### ISSUE TITLE
代码中有个公式不是很理解

### ISSUE THREAD

**Issue by a-daniu** [PRIS-CV/PCA-Net#2]
def co_att(feature1, feature2):
    B, N, W, H = feature1.shape
    x1 = feature1.reshape(B, N, W*H)
    x2 = feature2.reshape(B, N, W*H)
    I = -torch.bmm(x1, x2.permute(0,2,1))
    I = F.softmax(I,2)
    Y = torch.bmm(I, feature1.reshape(B, N, W*H)).reshape(B, N, W, H)
    Y = Y + feature1
    return Y
这段代码对应文中的哪些内容？

**Comment by dongliangchang** [comment]
@TIan1874

**Comment by TIan1874** [comment]
> def co_att(feature1, feature2):
> B, N, W, H = feature1.shape
> x1 = feature1.reshape(B, N, W_H) x2 = feature2.reshape(B, N, W_H)
> I = -torch.bmm(x1, x2.permute(0,2,1))
> I = F.softmax(I,2)
> Y = torch.bmm(I, feature1.reshape(B, N, W*H)).reshape(B, N, W, H)
> Y = Y + feature1
> return Y
> 这段代码对应文中的哪些内容？

对应文中公式(1)和公式(2)
